### ISSUE TITLE
Port all issues from domitry/nyaplot.

### ISSUE THREAD

**Issue by v0dro** [SciRuby/nyaplot#2]

**Comment by lokeshh** [comment]
I found this app which could be useful to port all issues automatically: https://github-issue-mover.appspot.com/

I just ported [one issue](https://github.com/SciRuby/nyaplot/issues/4) using this app.

Also it leaves a message mentioned to where the issue have been moved to as you can see [here](https://github.com/domitry/nyaplot/issues/1)

**Comment by v0dro** [comment]
Don't they have a script for porting all issues with comments?

**Comment by lokeshh** [comment]
I do not think they have a script to move all issues at once but it certainly does move one issue at a time with all its comments. [Here's](https://github.com/google/github-issue-mover) the code for this app. The comments mention whom are they originally from but they are posted by the user who gives authorization to the app (as you can see with [this issue](https://github.com/SciRuby/nyaplot/issues/4) that I ported).

I think I can write a script in Ruby that could port all the open issues or we can also do this manually. Its just 35 in number.

**Comment by v0dro** [comment]
Yeah you can write a script for porting issues when you have time. This is not priority right now, though.

PS: That script would be immensely useful to other people as well.

**Comment by lokeshh** [comment]
@v0dro its a bit complicated to write such a script as the website requires login before moving the issues. I was thinking maybe we can recreate this repo in the way daru was cloned to SciRuby/daru without losing any issues.

**Comment by v0dro** [comment]
Hmmm that makes sense. Let me ask Naoki.
